src/crypto/dtls.rs

```diff
@@ -59,12 +89,12 @@ impl DtlsCert {
     ///
     /// * **openssl** (defaults to on) for crypto backed by OpenSSL.
     /// * **wincrypto** for crypto backed by windows crypto.
-    pub fn new(p: CryptoProvider) -> Self {
+    pub fn new(p: CryptoProvider, dtls_cert_options: DtlsCertOptions) -> Self {
```

Comment:
Let's rename `dtls_cert_options` to `opts` and follow that through down. Let the type be the context.

Reply:
Done